### PR TITLE
{vis}[GCCcore/6.3.0] Updated X11 bundle and dependencies for GCCcore-6.3.0

### PR DESCRIPTION
**Pull request by geimer**
- Bumped components and dependencies to the latest available versions (fixes a number of CVEs, see https://lists.x.org/archives/xorg-announce/2016-October/002720.html)
 - Added libXfont2

### CODE REVIEW

**Comment by geimer** [comment]
Test report by @geimer
**FAILED**
Build succeeded for 14 out of 15 (15 easyconfigs in this PR)
swat - Linux debian 7.11, Intel(R) Xeon(R) CPU           X5550  @ 2.67GHz, Python 2.7.3
See https://gist.github.com/596320d121e169a54027e7b2668b6687 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 16 out of 16 (15 easyconfigs in this PR)
node2605.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3fca1dd0b7fb741dfeccdbf909e5e8ff for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 15 out of 15 (15 easyconfigs in this PR)
node2086.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fcf527e8a6ffab726fa8283ea585aa13 for a full test report.

File: easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'gettext'
+version = '0.19.8.1'
+
+homepage = 'http://www.gnu.org/software/gettext/'
+description = """GNU `gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
```
Comment (boegel) [inline easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-6.3.0.eb]: @geimer please get rid of backtick, cfr. #4146

File: easybuild/easyconfigs/g/gettext/gettext-0.19.8.1.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'gettext'
+version = '0.19.8.1'
+
+homepage = 'http://www.gnu.org/software/gettext/'
+description = """GNU `gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
```
Comment (boegel) [inline easybuild/easyconfigs/g/gettext/gettext-0.19.8.1.eb]: @geimer please get rid of backtick, cfr. #4146

**Comment by boegel** [comment]
lgtm, other than the trivial backtick remarks in the `gettext` description

@geimer any idea why the test report failed for you?

**Comment by geimer** [comment]
@boegel I have no clue.  In fact, I did a full build before trying with `--upload-test-report` and it worked fine.  I just started another build. Let's see what happens...

**Comment by geimer** [comment]
Test report by @geimer
**FAILED**
Build succeeded for 12 out of 15 (15 easyconfigs in this PR)
swat - Linux debian 7.11, Intel(R) Xeon(R) CPU           X5550  @ 2.67GHz, Python 2.7.3
See https://gist.github.com/37d245648c497311cacdd2be7adc23a4 for a full test report.

**Comment by boegel** [comment]
@geimer failed again, there's something weird going on here...

**Comment by geimer** [comment]
@boegel ???

Is it possible that if I run `eb --from-pr 4147 -f`, EasyBuild builds the packages in any order?  I see that `gettext` is built **before** `libxml2` although the latter is a dependency of the first...

**Comment by rjeschmi** [comment]
I have a similar problem with --from-pr and --upload-test-report I'll try to characterize the problem, but I feel like it has something to do with minimal toolchains (gut feeling, no real evidence)

cc: @ocaisa

**Comment by geimer** [comment]
@rjeschmi I don't have minimal toolchains enabled in my test env...

**Comment by rjeschmi** [comment]
@geimer well that is one thing eliminated then :)

It is certainly a dependency resolution issue and I also didn't see it except with --from-pr

**Comment by geimer** [comment]
@rjeschmi I see that @boegel also passed `-r`.  I guess this will eliminate the dependency thing?

**Comment by rjeschmi** [comment]
@geimer well dependency in the sense that in a particular PR there may be interdependencies where one needs to be built before the other.

**Comment by boegel** [comment]
@geimer not using `--robot` explains the issue, since the robot is responsible for ordering the builds

`--from-pr` without `--robot` will pull in all and build all easyconfigs, but build them in alphabetical order

**Comment by geimer** [comment]
Test report by @geimer
**SUCCESS**
Build succeeded for 15 out of 15 (15 easyconfigs in this PR)
swat - Linux debian 7.11, Intel(R) Xeon(R) CPU           X5550  @ 2.67GHz, Python 2.7.3
See https://gist.github.com/1decde7c27b1b5c72e346816683a07ed for a full test report.

**Comment by geimer** [comment]
@boegel There you go 😀

**Comment by rjeschmi** [comment]
Yeah, my problem is related to --robot too (my container build is eating some options somewhere so robot wasn't making it through).

**Comment by boegel** [comment]
Going in, thanks @geimer!

**Pull request merged**